### ISSUE TITLE
Use of transform must be qualified in JuliaDB

### ISSUE THREAD

**Issue by yakir12** [JuliaComputing/JuliaDB.jl#292]
When using `pushcol` I'm prompt to switch to the new API:
```julia
julia>     x = pushcol(x, :POI => POI.(x))
┌ Warning: `pushcol(t, args::Pair...)` is deprecated, use `transform(t, args...)` instead.
│   caller = top-level scope at none:0
```
but:
```julia
julia>     x = JuliaDB.transform(x, :POI => POI.(x))
WARNING: both OnlineStats and IndexedTables export "transform"; uses of it in module JuliaDB must be qualified
ERROR: UndefVarError: transform not defined
```

**Comment by joshday** [comment]
Ah, that's a conflict with LearnBase.  I'll change OnlineStats to not reexport LearnBase.

**Comment by jpsamaroo** [comment]
What's a full MWE for this? On JuliaDB master, using `pushcol(::IndexedTable, ::Pair)` doesn't result in a depwarn for me.

**Comment by yakir12** [comment]
I'm not on master.

**Comment by joshday** [comment]
Should be good to go with the new OnlineStats release!

**Comment by yakir12** [comment]
**TL;DR:** I had to install `JuliaDB#master` for this to actually work.

I'm still getting this warning:
```julia
WARNING: both OnlineStats and IndexedTables export "transform"; uses of it in module JuliaDB must be qualified
```
and sure enough it errors out with:
```julia
ERROR: UndefVarError: transform not defined
```
from trying this (which might be wrong API-wise - I don't actually know cause the docs don't have `transform` yet - but still):
```julia
x = JuliaDB.transform(x, :POI => POI.(x))
```
and sure enough, the culprit is:
```
(v1.1) pkg> st JuliaDB
    Status `~/.julia/environments/v1.1/Project.toml`
  [e7dc6d0d] DataValues v0.4.11
  [6deec6e2] IndexedTables v0.12.2
  [a93385a2] JuliaDB v0.12.0
  [a15396b6] OnlineStats v0.23.0
  [925886fa] OnlineStatsBase v0.10.2
  [2913bbd2] StatsBase v0.30.0
  [e0df1984] TextParse v0.9.1
  [10745b16] Statistics
```
That `OnlineStats` is stuck on v0.23.0 (while it's latest release is v0.25.0).

So I try to update and after a bit of local drama with my own packages (depending on JuliaDB) I'm stuck with:
```julia
(v1.1) pkg> add OnlineStats#master
  Updating git-repo `https://github.com/joshday/OnlineStats.jl.git`
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package JuliaDB [a93385a2]:
 JuliaDB [a93385a2] log:
 ├─possible versions are: [0.1.0-0.1.5, 0.2.0-0.2.3, 0.3.0-0.3.7, 0.4.0-0.4.4, 0.5.0, 0.6.0, 0.7.0-0.7.2, 0.8.0-0.8.5, 0.9.0, 0.10.0, 0.11.0-0.11.2, 0.12.0] or uninstalled
 ├─restricted to versions 0.12.0 by an explicit requirement, leaving only versions 0.12.0
 └─restricted by compatibility requirements with OnlineStats [a15396b6] to versions: [0.1.0-0.1.5, 0.2.0-0.2.3, 0.3.0-0.3.6] or uninstalled — no versions left
   └─OnlineStats [a15396b6] log:
     ├─possible versions are: 0.27.0 or uninstalled
     └─OnlineStats [a15396b6] is fixed to version 0.27.0
```
and only after updating to `JuliaDB#master` does this get resolved.
